go.mod

```diff
@@ -16,13 +16,14 @@ require (
 	github.com/zclconf/go-cty-yaml v1.0.2
 	google.golang.org/grpc v1.38.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.20.7
-	k8s.io/apimachinery v0.20.7
-	k8s.io/cli-runtime v0.20.7
-	k8s.io/client-go v0.20.7
-	k8s.io/kube-aggregator v0.20.7
-	k8s.io/kubectl v0.20.7
+	k8s.io/api v0.21.1
+	k8s.io/apimachinery v0.21.1
+	k8s.io/cli-runtime v0.21.1
+	k8s.io/client-go v0.21.1
+	k8s.io/kube-aggregator v0.21.1
+	k8s.io/kubectl v0.21.1
+	sigs.k8s.io/kustomize/api v0.8.10
 	sigs.k8s.io/yaml v1.2.0
 )
 
-//replace github.com/Azure/go-autorest v10.15.4+incompatible => github.com/Azure/go-autorest v13.0.0+incompatible
+replace github.com/tencentcloud/tencentcloud-sdk-go v3.0.82+incompatible => github.com/tencentcloud/tencentcloud-sdk-go v1.0.169
```

Comment:
This was required for me to be able to build at all, seems like AFAIK this project has deleted all its tags at some point in the past.